app/models/data_csv.rb

```diff
@@ -299,6 +305,7 @@ def self.initialize_with_weams
 
     names = Weam::USE_COLUMNS.map(&:to_s).join(', ')
 
+    # Include only those schools marked as approved (c.f., Weam model)
```

Comment:
Its the latin abbreviation for "conferre" meaning refer elsewhere (in this case, the Weam model) for more information.